apps/charterafrica/src/payload/fields/ecosystem/airtableBaseSelect.js

```diff
@@ -6,11 +6,11 @@ import useSWR from "swr";
 
 import fetchJson from "../../../utils/fetchJson";
 
-const baseUrl = `${process.env.PAYLOAD_PUBLIC_APP_URL}/api/v1/resources/ecosystem/schema?source=airtable&url=/meta/bases`;
+const baseUrl = `${process.env.PAYLOAD_PUBLIC_APP_URL}/api/v1/resources/ecosystem/bases`;
```

Comment:
```suggestion
const baseUrl = `${process.env.PAYLOAD_PUBLIC_APP_URL}/api/v1/resources/ecosystem/`;
```

Then basesUrl becomes baseUrl/bases, schemaUrl becomes baseUrl/schema?...